pageserver/src/tenant.rs

```diff
@@ -782,7 +782,7 @@ impl Tenant {
         let tenant_conf = match Self::load_tenant_config(conf, tenant_id) {
             Ok(conf) => conf,
             Err(e) => {
-                error!("load tenant config failed: {}", e);
+                error!("load tenant config failed: {:?}", e);
```

Comment:
This is to get full error message in the log, if parsing the config file fails. I had a bug in the boolean-parsing code during development, and it was hard to debug because you got a very generic error message.